app/views/posts/_post_table.html.erb

```diff
@@ -0,0 +1,120 @@
+				    <!-- for now do not show the cancelled requests -->
+						<% post_needs = p.post_needs %>
+						<tr>
+							<td>
+								<p class="mobile-only table-heading">Posted By</p>
+								<%= link_to "#{p.poster.first_name} #{p.poster.last_name}", user_path(p.poster) %></br>
+								<small><%= format_time(p.date_posted.to_time) %></small>
+							</td>
+							<td colspan="1">
+								<p class="mobile-only table-heading">Address</p>
+								<%= p.street_1 %><br/>
+								<% unless p.street_2.empty? %>
+									<%= p.street_2 %><br/>
+								<% end %>
+								<%= p.city %>, <%= p.state %> <%= p.zip%>
+							</td>
+							<td>
+								<p class="mobile-only table-heading">Post Needs</p>
+								<ul>
+									<% post_needs.each do |pn| %>
+										<li>
+											<%= pn.need.name %>
+											<% if pn.complete? %>
+												✓
+											<% end %>
+										</li>
+									<% end %>
+								</ul>
+								
+							</td>
+							<td>
+								<p class="mobile-only table-heading">Status</p>
+								<%  if p.date_cancelled.nil? and p.date_claimed.nil? %>
+											<div class = "red-text">Unclaimed</div>
+							    
+								<%elsif !p.date_completed.nil? %>
+										<div class = "green-text">	Completed on <%= p.date_completed.to_date %></div>
+								
+								<% 	elsif p.date_claimed %>
+											Claimed by <%= link_to p.claimer.name, user_path(p.claimer) %> on <%= p.date_claimed.to_s %>
+											</br>
+								<%	elsif p.date_cancelled %>
+											Cancelled on <%= p.date_cancelled.to_s %>
+
+								<% end %>
+								<% if p.claimer_id %>
+									<p class="mobile-only table-heading">Claimer</p>
+									<% if logged_in? and p.claimer_id == current_user.id %>
+										<%if p.date_completed.nil?%><%= link_to 'Unclaim Request', unclaim_request_path(id: p.id),data: {confirm: "Are you sure you want to cancel this request?"}%>
+										<%end%>
+									<% end %>
+								<% elsif logged_in? and p.date_cancelled.nil? %>
+									<%= link_to 'Claim Request', claim_request_path(id: p.id) %>
+								<% end %>
+							</td>
+							<td>
+								<p class="mobile-only table-heading">Actions</p>
+
+								<%if !p.date_completed.nil? or p.claimer_id.nil? or p.claimer_id!=current_user.id%>
+
+								<div id ="viewpost"><%= link_to "View Post", post_path(p) %></div>
+
+								<%else%>
+
+								<%= link_to "Complete Needs", post_path(p) %>
+
+								<%end%>
+
+								<% if logged_in? and current_user.id == p.poster_id and p.date_cancelled.nil? %>
+									<p class="mobile-only table-heading">Actions</p>
+									<small><%= link_to 'Cancel Request', cancel_request_path(id: p.id), data: {confirm: "Are you sure you want to cancel this request?"}%>
+									
+									</small>
+
+								<% end %>
+							</td>
+						</tr>
+					<%end%>
+				<% end %>
+				<%= will_paginate @posts %>
+			</tbody>
+		</table>
+		<%will_paginate @posts%>
+	<% else %>
+		<p><i>There are no posts to display now.</i></p>
```

Comment:
New feature:
For mobile view: only the title will be shown. click view_details for more detials